cmd/executor/echo/main.go

```diff
@@ -29,7 +34,8 @@ type EchoExecutor struct{}
 func (EchoExecutor) Metadata(context.Context) (api.MetadataOutput, error) {
 	return api.MetadataOutput{
 		Version:     version,
-		Description: "Echo is an example Botkube executor plugin used during e2e tests. It's not meant for production usage.",
+		Description: description,
+		JSONSchema:  jsonSchema(),
```

Comment:
as stated in the AC in task:
> - Make sure that JSON schema input property is extensible, so in the future we can accept not only inline JSON schema but also external refs.
>    - it will allow us to put size restriction on inlined JSON Schema, so bigger will be hosted externally.

with such approach is not possible, better is to do:
```
JSONSchema:
  Value:  "string"
```
so later we can have:
```
JSONSchema:
  RefURL:  "http://example.com/schema.json"
```